Documentation/configure-kubectl.md

```diff
@@ -1,16 +1,17 @@
-## Configure kubectl
+# Setting up `kubectl`
 
-The primary CLI tool used to interact with the Kubernetes API is called `kubectl`.
+`kubectl` is a command-line program for interacting with the Kubernetes API. The following steps should be done from a local workstation to configure `kubectl` to work with a new cluster.
 
-The following steps should be done from your local workstation to configure `kubectl` to work with your new cluster.
+## Download the `kubectl` Executable
 
-First, download the binary using a command-line tool such as `wget` or `curl`:
+`kubectl` is not yet widely packaged by distributions, so the current practice is to download it directly from the Kubernetes release artifact site with a tool like `wget` or `curl`. The linux `kubectl` binary can be fetched with a command like:
 
 ```sh
-# Replace ${ARCH} with "linux" or "darwin" based on your workstation operating system
-$ ARCH=linux; wget https://storage.googleapis.com/kubernetes-release/release/v1.1.2/bin/${ARCH}/amd64/kubectl
+wget https://storage.googleapis.com/kubernetes-release/release/v1.1.2/bin/linux/amd64/kubectl
```

Comment:
What do you think about this:

1. Switch to curl since both CoreOS and OS X don't have wget by default
2. Have the full commands for both platforms ready for copy/paste

```sh
# for Linux
curl https://storage.googleapis.com/kubernetes-release/release/v1.1.2/bin/linux/amd64/kubectl > kubectl

# for OS X
curl https://storage.googleapis.com/kubernetes-release/release/v1.1.2/bin/darwin/amd64/kubectl > kubectl
```